src/main/java/org/opennms/e2e/stacks/OpenNMSHelmOCEStack.java

```diff
@@ -0,0 +1,195 @@
+        ImmutableMap.Builder<String, Function<GizmoDockerStacker, ContainerConfig>> containers =
+                new ImmutableMap.Builder<>();
+        containers.putAll(super.getContainersByAlias());
+
+        if (redundant) {
+            redundanctOCEs.forEach(oceAlias -> containers.put(oceAlias, this::oceConfig));
+        } else {
+            containers.put(OCE, this::oceConfig);
+        }
+
+        return containers.build();
+    }
+
+    @Override
+    public List<Consumer<GizmoDockerStacker>> getWaitingRules() {
+        ImmutableList.Builder<Consumer<GizmoDockerStacker>> waitingRules = new ImmutableList.Builder<>();
+        waitingRules.addAll(super.getWaitingRules());
+
+        if (redundant) {
+            redundanctOCEs.forEach(oceAlias -> waitingRules.add(stacker -> waitForOCEByAlias(oceAlias, stacker)));
+        } else {
+            waitingRules.add(stacker -> waitForOCEByAlias(OCE, stacker));
+        }
+
+        return waitingRules.build();
+    }
+
+    @SuppressWarnings("ResultOfMethodCallIgnored")
+    private ContainerConfig oceConfig(GizmoDockerStacker stacker) {
+        Path sentinelOverlayPath = setupOverlay("sentinel-overlay");
+        Path deployPath = sentinelOverlayPath.resolve("deploy");
+        Path standaloneFeatures = deployPath.resolve("features-standalone.xml");
+        Path redundantFeatures = deployPath.resolve("features-redundant.xml");
+
+        if (redundant) {
+            standaloneFeatures.toFile().delete();
+            redundantFeatures.toFile().renameTo(deployPath.resolve("features.xml").toFile());            
+            insertApplicationId(sentinelOverlayPath);
+        } else {
+            redundantFeatures.toFile().delete();
+            standaloneFeatures.toFile().renameTo(deployPath.resolve("features.xml").toFile());
+        }
+
+        return ContainerConfig.builder()
+                .image("opennms/sentinel:oce")
+                .exposedPorts("8301/tcp")
+                .env("KARAF_DEBUG_LOGGING=org.opennms.oce")
+                .hostConfig(HostConfig.builder()
+                        .publishAllPorts(true)
+                        .autoRemove(true)
+                        .binds(sentinelOverlayPath + ":/opt/sentinel-overlay")
+                        .links(String.format("%s:kafka", stacker.getContainerInfo(KafkaStack.KAFKA).name()))
+                        .build())
+                .cmd("-f")
+                .build();
+    }
+
+    private void insertApplicationId(Path overlayDir) {
+        String applicationIdProperty = "\napplication.id = oce-datasource-instance-" + instanceNum++;
+
+        try {
+            Files.write(Paths.get(overlayDir.toString(), "etc", "org.opennms.oce.datasource.opennms.kafka" +
+                    ".streams.cfg"), applicationIdProperty.getBytes(), StandardOpenOption.APPEND);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Can we rethrow as a RuntimeException instead of just printing the stack trace? There are a few other cases of this as well.